docs/codeql/codeql-language-guides/customizing-library-models-for-java.rst

```diff
@@ -0,0 +1,417 @@
+.. _customizing-library-models-for-java:
+
+:orphan:
+:nosearch:
+
+Customizing Library Models for Java
+===================================
+
+.. include:: ../reusables/beta-note-customizing-library-models.rst
+
+The Java analysis can be customized by adding library models (summaries, sinks and sources) in data extension files.
```

Comment:
Are the terms summary, sink, source known to the target audience? I guess the answer is 'yes', just to confirm.

Reply:
This is a good point and there is a similar comment below.
I will elaborate a bit more!